### PR TITLE
update marketo country and state field and js

### LINKED ISSUE #851: Standardise form implementations
As mentioned by @anthonydillon in #846:

> I want to bring up the number of forms across the site and the different methods they have been implemented. I think as they seem to make up such a large proportion of the site need to develop a elegant and consistent method.

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* rewrote marketo's js that shows/hides the state/province fields based on what you select in the country select dropdown
* added a shared _state.html file as not all forms require state
* removed the old js from the pages
* added country to the /download/cloud ebook form per Sales (Shelley) request as phone numbers often don't have a country code

## QA

1. check that /static/js/forms.js looks ok
2. check that these forms all work as expected
- [x] check /about/contact-us/form
- [x] check /cloud/contact-us
- [x] check /cloud/openstack/managed-cloud/contact-us
- [x] check /cloud/training/onsite-contact-us
- [x] check /cloud/training/server-admin-onsite-contact-us
- [x] check /desktop/tenders-contact-us
- [x] check /download/cloud
- [x] check /internet-of-things/contact-us
- [x] check /internet-of-things/digital-signage/contact-us
- [x] check /management/contact-us
- [x] check /server/contact-us
- [x] check /tablet/contact-us

## Issue / Card

Partly fixes #851

### CODE REVIEW

File: templates/download/shared/_get_ebook.html [hunk 1]
```diff
@@ -36,6 +36,8 @@
                             <input required  id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField  mktoRequired" >
                         </li>
 
+                        {% include "shared/forms/_country.html" %}
```
Comment (anthonydillon) [inline templates/download/shared/_get_ebook.html]: Do you mean to add country. This adds country to /download/cloud

Comment (pmahnke) [inline templates/download/shared/_get_ebook.html]: yes, and country only.

**Comment by pmahnke** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;sTBmQV52&#x2F;1746-add-a-country-field-to-the-download-cloud-ebook-form"> add a country field to the &#x2F;download&#x2F;cloud ebook form</a>

File: templates/shared/forms/_state.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+        </optgroup>
+    </select>
+</li>
+
+<script src="/static/js/forms.js"></script>
```
Comment (anthonydillon) [inline templates/shared/forms/_state.html]: Nice idea to include it here. Stops forgetting.

**Pull request merged**